event-handlers/Emergence/People/RegistrationRequestHandler/registerComplete/listshine.php

```diff
@@ -0,0 +1,9 @@
+<?php
+//this file should be in /event-handlers/People/RegistretationRequestHandler/registerComplete/  directory of your site
+if(isset($_POST['Subscribe'])){
```

Comment:
@MPorcic is there a registration page change that goes along with this to add a subscribe checkbox? I'm considering adding a standard Newsletter on/off option to profile and registration that any hookins could use